src/rules/comment-empty-line-before/__tests__/index.js

```diff
@@ -73,3 +74,17 @@ testRule("never", tr => {
   tr.notOk("a {}\r\n\r\n\r\n/** comment */", messages.rejected, "CRLF")
   tr.notOk("a { color: pink;\n\n/** comment */\ntop: 0; }", messages.rejected)
 })
+
+const testRuleScss = ruleTester(rule, ruleName, {
+  postcssOptions: {
+    syntax: scssSyntax,
+  },
+})
+
+testRuleScss("always", tr => {
+  tr.ok("a { color: pink;\n// comment\ntop: 0; }", "line comment ignored")
+})
+
+testRuleScss("never", tr => {
+  tr.ok("a { color: pink;\n\n// comment\ntop: 0; }", "line comment ignored")
+})
```

Comment:
Since the intention in this one was to allow multiple lines of inline comments, let's add another test for that, e.g. 

```css
// first line
// second line
a { color: pink; }
```